src/templates/devinfo.handlebars

```diff
@@ -160,10 +160,10 @@
         {{/ne_and}}
     {{/if}}
     </h3>
-    <ul>
+    <ul id="contributors--readonly" class="userpage-link">
```

Comment:
Why is `userpage-link` on the parent element of the link? Why not put it directly on the `<a>`?